apps/web/modules/videos/views/videos-single-view.getServerSideProps.tsx

```diff
@@ -103,12 +104,18 @@ export async function getServerSideProps(context: GetServerSidePropsContext) {
 
   const session = await getServerSession({ req });
 
-  // set meetingPassword to null for guests
+  // set meetingPassword for guests
   if (session?.user.id !== bookingObj.user?.id) {
+    const videoReference = getCalVideoReference(bookingObj.references);
+    const guestMeetingPassword = await generateGuestMeetingTokenFromOwnerMeetingToken(
+      videoReference.meetingPassword
+    );
```

Comment:
This is required because we want a meeting token with `enable_recording_ui: false` so that default recording button doesn't show up